rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/api/BakeRequest.groovy

```diff
@@ -64,6 +64,8 @@ class BakeRequest {
   String template_file_name
   @ApiModelProperty("A map of key/value pairs to add to the packer command")
   Map extended_attributes
+  @ApiModelProperty("The name of a json file containing key/value pairs to add to the packer command")
```

Comment:
Oh, actually looks like it's an absolute path. I guess that should be documented instead?